Dockerfile

```diff
@@ -1,6 +1,10 @@
-FROM progrium/busybox:latest
+FROM gliderlabs/alpine:3.6
 
-MAINTAINER Bjoern Gruening <bjoern.gruening@gmail.com>
+LABEL author="Roberto Spreafico <roberto.spreafico@gmail.com>"
```

Comment:
Can we put this into the readme file?

Reply:
Sorry for the delay, I missed the notification. Sure thing, and I do not have strong feelings about retaining this information anyway, it's a very simple image. Feel free to delete.